pkg/vsphere/tasks/waiter.go

```diff
@@ -56,20 +57,21 @@ func Wait(ctx context.Context, f func(context.Context) (Task, error)) error {
 //       return vm, vm.Reconfigure(ctx, config)
 //    })
 func WaitForResult(ctx context.Context, f func(context.Context) (Task, error)) (*types.TaskInfo, error) {
-	var err error
 	var info *types.TaskInfo
 	var backoffFactor int64 = 1
 
 	for {
-		var t Task
-		if t, err = f(ctx); err == nil {
+		t, err := f(ctx)
+		log.Debugf("New task: %s", taskMoid(t))
```

Comment:
and i had that in my *original* pr!  i squashed and merged and it went away!  It was it's own commit, was in the PR (i made sure the 3 lines were there) and poof, gone.  AND!  The original PR (since i reused the same branch) now includes these changes.  PRs apparently aren't immutable!  

Why are we using this broken system again?  If PRs aren't immutable, we can't rely on review history.